src/app/submissions/stub/submission.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @name submissionParse
+ * @desc Fetch the submission table
+ * @param {Function} func - Function to execute after fetching & mapping done
+ * @return {Promise} a `fetch()` `Promise`
+ * @author minhducsun2002
+ */
+
+async function submissionParse(func) {
+    return fetch(
+        `http://${window.location.hostname}:${window.location.port}/api/subs`
```

Comment:
Shorten this to "/api/subs"

Reply:
Address in https://github.com/vnma0/hestia/pull/19/commits/84c0282afababde0e3ad70cf90345d947d22553c.